phonecall_plan/__openerp__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 6IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Phonecall plan',
+    'summary': "Wizard to plan calls from leads/partners",
+    'version': '8.0.1.0.0',
+    'depends': ['base', 'crm'],
```

Comment:
Nothing wrong here but you can improve it putting the dependencies and data files in a separate line (check other OCA modules)
Non-blocking though